android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.kt

```diff
@@ -400,23 +416,39 @@ open class GestureHandler<ConcreteGestureHandlerT : GestureHandler<ConcreteGestu
     dispatchTouchEvent()
   }
 
-  private fun addPointerData(pointerData: PointerData) {
-    if (touchEventPayload == null) {
-      touchEventPayload = Arguments.createArray()
+  private fun addChangedPointer(pointerData: PointerData) {
+    if (changedTouchesPayload == null) {
+      changedTouchesPayload = Arguments.createArray()
     }
 
-    touchEventPayload?.pushMap(Arguments.createMap().apply {
-      putInt("id", pointerData.pointerId)
-      putDouble("x", PixelUtil.toDIPFromPixel(pointerData.x).toDouble())
-      putDouble("y", PixelUtil.toDIPFromPixel(pointerData.y).toDouble())
-      putDouble("absoluteX", PixelUtil.toDIPFromPixel(pointerData.absoluteX).toDouble())
-      putDouble("absoluteY", PixelUtil.toDIPFromPixel(pointerData.absoluteY).toDouble())
-    })
+    changedTouchesPayload?.pushMap(createPointerData(pointerData))
+  }
+
+  private fun addPointerToAll(pointerData: PointerData) {
+    if (allTouchesPayload == null) {
+      allTouchesPayload = Arguments.createArray()
+    }
+
+    allTouchesPayload?.pushMap(createPointerData(pointerData))
```

Comment:
Shouldn't `allTouchesPayload` always exist here? Same with `changedTouchesPayload`.

Reply:
Yeah, you're right. Fixed in https://github.com/software-mansion/react-native-gesture-handler/pull/1732/commits/87ed53cb7b3f3ffe410712e48eea20ac11caa38d.